### DISCUSSION TITLE
Comments have too much padding at the bottom

### DISCUSSION THREAD

**Discussion by mouzedrift** [ppy/osu#22360] [Q&A] [answered]
osu!lazer:
![image](https://user-images.githubusercontent.com/43358824/213940530-ff8384eb-48ad-42ab-897d-0e1842746762.png)

osu!web:
![image](https://user-images.githubusercontent.com/43358824/213940483-ff222fbd-ed3b-42ba-b482-c2e365c74ca8.png)

**Comment by Joehuu** [comment, marked as answer]
Fixed in https://github.com/ppy/osu/pull/22507.
